corehq/apps/domain/views/releases.py

```diff
@@ -187,6 +189,8 @@ def update_release_restriction(request, domain, restriction_id, active):
     return JsonResponse(data=response_content)
 
 
+@login_and_domain_required
```

Comment:
`@login_and_domain_required` seems overbroad.  Looks like the `release_build` view uses `@require_can_edit_apps`, which seems more appropriate.

Reply:
this i think would be more used by the rollout team which might not have that permission to edit apps. The correct permission might actually be `manage_releases` but i would need to check with team about that.
I will update to `require_can_edit_apps` for now and then do a follow up on the best possible permission

Reply:
https://github.com/dimagi/commcare-hq/pull/24692/commits/6a64a61ad1474f1d0733ee8d9a0a1c7c4fd74739